app/ide-desktop/lib/dashboard/src/providers/AuthProvider.tsx

```diff
@@ -368,7 +368,13 @@ export default function AuthProvider(props: AuthProviderProps) {
           document.cookie = `logged_in=yes;max-age=34560000;domain=${parentDomain};samesite=strict;secure`
 
           // Save access token so can it be reused by the backend.
-          cognito.saveAccessToken(session.accessToken)
+          cognito.saveAccessToken({
+            accessToken: session.accessToken,
+            clientId: session.clientId,
+            expireAt: session.expireAt,
+            refreshToken: session.refreshToken,
+            refreshUri: session.refreshUrl,
```

Comment:
is there a reason why this specific field is different (causing the need for this object literal?)

(i suppose `email` isn't supposed to be in the file, so you might need the object literal anyway...)